src/css/start-script-generator.css

```diff
@@ -0,0 +1,198 @@
+.server-config-container {
+  background-color: #222;
+  border-radius: 10px;
+  padding: 20px;
+  color: #fff;
+  width: 100%;
+}
+
+.config-section {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+}
+
+.filename-input {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 15px;
+  height: 25px;
+  width: 40%;
+  padding: 20px 20px 20px 10px;
+}
+
+textarea {
```

Comment:
idk how we do styles for the docs, but having random global tag selectors like this is kinda meh
can't we use scss and nest everything under .server-config-container to prevent leaking of styles?